packages/core/src/access.ts

```diff
@@ -90,35 +126,67 @@ export function canWriteResource(accessPolicy: AccessPolicy, resource: Resource)
  * @param resource The resource.
  * @param readonlyMode True if the resource is being read.
  * @returns True if the resource matches the access policy.
+ * @deprecated Use satisfiedAccessPolicy() instead.
  */
 export function matchesAccessPolicy(accessPolicy: AccessPolicy, resource: Resource, readonlyMode: boolean): boolean {
   if (accessPolicy.resource) {
     for (const resourcePolicy of accessPolicy.resource) {
-      if (matchesAccessPolicyResourcePolicy(resource, resourcePolicy, readonlyMode)) {
+      if (
+        matchesAccessPolicyResourcePolicy(
+          resource,
+          readonlyMode ? AccessPolicyInteraction.READ : AccessPolicyInteraction.UPDATE,
+          resourcePolicy
+        )
+      ) {
         return true;
       }
     }
   }
   return false;
 }
 
+/**
+ * Check that there is an access policy permitting the given resource interaction, returning the matching policy object.
```

Comment:
Super minor nit:  When writing jsdoc comments, I try to follow the recommended javadoc convention:

> Use 3rd person (descriptive) not 2nd person (prescriptive). The description is in 3rd person declarative rather than 2nd person imperative.
> 
> Gets the label. (preferred)
> 
> Get the label. (avoid)

https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#styleguide

Reply:
Cool, that makes sense to me!